uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedItem.kt

```diff
@@ -82,7 +81,7 @@ fun FeedItem(
                 }
                 .width(96.dp)
 //                .aspectRatio(16F / 9F)
-                .aspectRatio(1F / 1F).border(1.dp, Color.Black),
+                .aspectRatio(1F / 1F),
```

Comment:
.clip(shape = RoundedCornerShape(4.dp)) to apply rounded corners.
*I took a liberty with the dp size as 2px is not easily translatable to dx.

Reply:
Certainly the figma design is rounded👀

Reply:
@mightyfrog 
Thanks for the good advice!

@takahirom 
The corners are not rounded.
Do you want to apply what he suggested?

Reply:
Since the purpose of the following issue is to show it to the designer, I think that it is better to arrange the appearance, and for that purpose it seems better to round it.
https://github.com/DroidKaigi/conference-app-2021/issues/88
It's okay to do the rounding with this PR, or with another PR. Please choose the one you like.

Reply:
It looks like there are a few things we need to consider for the design.
It would be better to change the roundness after discussing it in #88.

Reply:
OK! I will merge this PR! Thank you for thinking!

Reply:
Thank you, too.